### ISSUE TITLE
{Guidance} to speedup Qwen image model 2.1 in intel irisx iGPU

### ISSUE THREAD

**Issue by ak-sangal** [huggingface/optimum-intel#2011]
as i am using qwen image model in text to image and image to image edit using int4 model based on below 2 scripts, i am using intel irisx iGPU with 16 GB RAM guide me how speed up in igpu

script below 
import torch
from optimum.intel import OVQwenImage21Pipeline

MODEL_DIR = "qwen-image-2.1-ov-int4"
SEED = 42

pipe = OVQwenImage21Pipeline.from_pretrained(MODEL_DIR)
pipe.to("GPU")

prompt = "A cat holding a sign that says hello world"
image = pipe(
    prompt=prompt,
    negative_prompt=" ",
    true_cfg_scale=4.0,
    num_inference_steps=50,
    height=512,
    width=512,
    output_resolution=512,
    generator=torch.Generator("cpu").manual_seed(SEED),
).images[0]

import torch
from diffusers.utils import load_image
from optimum.intel import OVQwenImage21Img2ImgPipeline

MODEL_DIR = "qwen-image-2.1-ov-int4"
SEED = 21

pipe = OVQwenImage21Img2ImgPipeline.from_pretrained(MODEL_DIR)
pipe.to("GPU")

init_image = load_image("qwen_image21_ov_t2i.png")

prompt = "Add a red party hat on the cat"
image = pipe(
    prompt=prompt,
    image=init_image,
    negative_prompt=" ",
    true_cfg_scale=1.0,
    num_inference_steps=20,
    height=512,
    width=512,
    output_resolution=512,
    generator=torch.Generator("cpu").manual_seed(SEED),
).images[0]
image.save("qwen_image21_ov_edit.png")
print("saved qwen_image21_ov_edit.png", image.size)

image.save("qwen_image21_ov_t2i.png")
print("saved qwen_image21_ov_t2i.png", image.size)
